src/library/Uno.Material/Styles/Controls/v2/PipsPager.Base.xaml

```diff
@@ -382,14 +382,6 @@
 				Value="{StaticResource MaterialPipsPagerPreviousPageButtonStyle}" />
 		<Setter Property="NextButtonStyle"
 				Value="{StaticResource MaterialPipsPagerNextPageButtonStyle}" />
-		<!-- Uno already implemented the 2.8.0 MUX version of the control -->
-		<!-- Uno.Themes reference is still on the MUX v2.6.0-prerelease.210113001 version for now -->
-		<!-- Remove this file, PipsPager.UWP.xaml,and use <Setter Property=NormalPipStyle /> and <Setter Property=SelectedPipStyle /> only in PipsPager.xaml -->
-		<!-- with the rest of the code, once we can bump the Microsoft.UI.Xaml version -->
-		<not_win:Setter Property="SelectedPipStyle"
-						Value="{StaticResource MaterialPipsPagerSelectedPipButtonStyle}" />
-		<not_win:Setter Property="NormalPipStyle"
-						Value="{StaticResource MaterialPipsPagerNormalPipButtonStyle}" />
```

Comment:
This code is in `BaseMaterialPipsPagerStyle` which is the common code. We can't determine which property to use here. So we have `PipsPager.UWP.xaml` and `PipsPager.xaml` (better if it's named `PipsPager.WinUI.xaml` for clarity actually)

- `PipsPager.UWP.xaml` defines `MaterialPipsPagerStyle` based on `BaseMaterialPipsPagerStyle` and chooses the property based on `win` vs `not_win` (Uno)

`PipsPager.xaml` (which is actually WinUI-flavor only) defines `MaterialPipsPagerStyle` based on `BaseMaterialPipsPagerStyle` and always uses the newer `SelectedPipStyle` and `NormalPipStyle` properties as they are available both on Windows builds and Uno.